### DISCUSSION TITLE
Different glyphs used for 3 and 9 in Inter used in Blender 4.0

### DISCUSSION THREAD

**Discussion by mceh** [rsms/inter#618] [Q&A] [answered]
Hello,
as you know, open source 3D software Blender uses Inter as it’s UI font since version 4.0. As a designer myself I immediately recognized the much more readable and cleaner numbers in Blender, although they didn’t resemble the Inter I know.

The „3“ has a flat top and the number „9“ a straight stem for example. I couldn't find any other version of Inter with these features. The version used in Blender is reported as 3.019 in Finder. 
Versions on Googlefonts as well as the current versions of Inter 3 and 4 on Github don't seem to incorporate these glyph forms. Googling didn't help either.

I wonder what's going on here: Did the Blender people change those glyphs and made their own version of Inter or what happened? I would like a full stack of this version for web design purposes.

Hope you have an answer! I’m super curious about this :-D

Thanks
Markus

**Comment by kenmcd** [comment, marked as answer]
Sounds like they have enabled Stylistic Set 1 (`ss01`).

![SS01-2023-10-12_13-46-18](https://github.com/rsms/inter/assets/844920/92d03f5c-a473-428f-a448-a928771d6699)

Take a look at the features available here: https://rsms.me/inter/#features

**Reply by mceh** [reply]
Thank you. Yes, that’s exactly it!
